java/org/contikios/cooja/MoteInterfaceHandler.java

```diff
@@ -254,6 +256,23 @@ public Radio getRadio() {
     return myRadio;
   }
 
+  /**
+   * Returns the first serial port interface (if any).
+   *
+   * @return Serial port interface
+   */
+  public SerialPort getSerialPort() {
+    if (mySerialPort == null) {
+      for (var moteInterface : moteInterfaces) {
+        if (moteInterface instanceof SerialPort port) {
+          mySerialPort = port;
+          break;
+        }
+      }
+    }
```

Comment:
This can be simplified by using `getInterfaceOfType()` to lookup the serial port.

Reply:
I tried, but that doesn't let me pass in `SerialPort.class`, I think it is because it is an interface.

Reply:
Ah, SerialPort is not a MoteInterface. I missed that. Sorry.